doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md

```diff
@@ -6,8 +6,8 @@
 # SECURITY SERVER USER GUIDE
 **X-ROAD 6**
 
-Version: 2.15
-20.02.2017
+Version: 2.16
```

Comment:
There should be a couple of empty spaces after the version number for the rows to wrap correctly. See https://github.com/vrk-kpa/X-Road/blob/de9adefe331074eaf20f3da42e676df97b300a4b/doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md